fetch-npm-node.js

```diff
@@ -1,16 +1,21 @@
 "use strict";
 
-var realFetch = require('node-fetch');
-module.exports = function(url, options) {
-	if (/^\/\//.test(url)) {
```

Comment:
Maybe you disagree with why this was added — so that the client side and server side interface was the same (you can make requests to `//domain.com` and both would work) but I don't think we need to remove this at this stage — would require a major version bump.